backend/digitales_warten.py

```diff
@@ -99,17 +99,27 @@ def delete_queue(place_id, queue_id):
     db.session.commit()
     return ''
 
+@app.route('/places/<place_id>/queues/<queue_id>', methods=['POST'])
+def add_entry(place_id, queue_id):
+    place = Place.query.filter_by(id=place_id).first()
+    if place is None:
+        abort(404)
+    queue = Queue.query.filter_by(id=queue_id).filter_by(place=place).first()
+    if queue is None:
+        abort(404)
+    data = request.json
+    if 'name' not in data:
+        abort(400)
+    entry_name = data['name']
+    entry_id = generate_entry_id(entry_name)
+    ticket_number = Slot.query.filter_by(queue=queue).count() + 1
```

Comment:
This does not work, because removing users reduces count

Reply:
Done